.codeclimate.yml

```diff
@@ -1,8 +1,34 @@
 engines:
   pep8:
     enabled: true
+
+  csslint:
+    enabled: true
+
+  duplication:
+    enabled: true
+    config:
+      languages:
+        - javascript:
+        - python:
+
+  eslint:
+    enabled: true
+
+  fixme:
+    enabled: true
+
+  markdownlint:
+    enabled: true
+
 ratings:
-   paths:
-   - "**.py"
-exclude_paths:     
-  - jarbas/core/migrations/
+  paths:
+    - "**.css"
+    - "**.js"
+    - "**.md"
+    - "**.py"
+    - "**.sh"
+
+exclude_paths:
```

Comment:
I found my self once not following some heading rules, because they didn't look as I wanted. And I see we doing the same some time in ours  markdowns. But as said before _if it is better for the project I'll be okay with it =D_